tests/model_registry/model_catalog/conftest.py

```
@@ -59,3 +61,21 @@ def updated_catalog_config_map(
 @pytest.fixture(scope="class")
 def expected_catalog_values(request: pytest.FixtureRequest) -> dict[str, str]:
     return request.param
+
+
+@pytest.fixture(scope="function")
+def updated_catalog_config_map_add_model(
+    request: pytest.FixtureRequest,
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)